.github/workflows/ci.yaml

```diff
@@ -0,0 +1,240 @@
+name: GitHub Actions CI
+
+# Cache is used for all Windows and macOS dependencies (size approx. 1230 * 2 + 1850 = 4310 MiB)
+# Cache is not used for Ubuntu builds, because it already has all dependencies except
+# the appropriate libtorrent version, which only takes 3-5 minutes to build from source anyway
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    types: [edited, opened, reopened, synchronize]
+    branches: [ master ]
+
+env:
+  # Qt: 5.15.0
+  # libtorrent: RC_1_2 HEAD, 1.2.10
+  VCPKG_COMMIT: 32eccc18191fbb57b159784a1724d2d00613ae82
```

Comment:
@Chocobo1 

Same reasoning as https://github.com/qbittorrent/qBittorrent/pull/13288#discussion_r491113329.

>That's not good practice, it has the potential to break our CI and will be difficult to track down - one would have to check recent updates to _all_ actions to figure out what changed. It is preferable to bump these every now and then. The CI itself will catch any breakage due to specific version bumps, and the whole situation becomes much easier to investigate and resolve, since we'll know immediately what changed.

Especially because currently, vcpkg package versions depend on the vcpkg version itself. This is expected to change int the near future though, when they implement versioning support.

Reply:
For the record: https://github.com/qbittorrent/qBittorrent/pull/13288#discussion_r491158732